src/_content/posts/call-for-2025-volunteers.md

```diff
@@ -0,0 +1,301 @@
+---
+author: DjangoCon US Communications Team
+category: General
+date: 204-10-18 06:00:00
+layout: post
+title: "Call for Volunteers: DjangoCon US 2025"
+---
+
+As our program chair, Drew Winstel, mentioned in his opening remarks today, we would love to have more volunteers for DjangoCon US 2025. Expect to hear details on dates and venues within the next month or so.
```

Comment:
If @PeterGrand is happy to plug this during the Wednesday opening remarks, put him in here instead of me. Also maybe be a little more vague about when we'll announce the conference

Reply:
I'm going to remove this paragraph entirely. Peter's got a lot on his plate and I'd rather let him fit things in as needed. The callout doesn't do much in my opinion.